src/build_function.jl

```diff
@@ -57,7 +63,7 @@ end
 
 function destructure_arg(arg::Union{AbstractArray, Tuple}, inbounds, name)
     if !(arg isa Arr)
-        DestructuredArgs(map(unwrap, arg), name, inbounds=inbounds)
+        DestructuredArgs(map(unwrap, arg), name, inbounds=inbounds, create_bindings=false)
```

Comment:
Should we really do that by default? It will make a lot of models slower.

Reply:
Wouldn't `let` make compilation slower?

Reply:
You can choose to not lower it into `let` but just assignments.

Reply:
Oh i see... i think there's an option for that too 😅